monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/JobTriggeringController.java

```diff
@@ -49,19 +55,37 @@
     @Autowired
     SchedulingService _schedulingService;
 
-    @RequestMapping(produces = "application/json")
+    @RequestMapping(method = { RequestMethod.DELETE, RequestMethod.GET, RequestMethod.HEAD, RequestMethod.OPTIONS,
+            RequestMethod.PATCH, RequestMethod.PUT, RequestMethod.TRACE }, produces = "application/json")
     @ResponseBody
     @RolesAllowed(SecurityRoles.SCHEDULE_EDITOR)
     public Map<String, String> invokeJob(@PathVariable("tenant") final String tenant,
             @PathVariable("job") String jobName, @RequestParam(value = "block", required = false) Boolean block,
             @RequestParam(value = "timeoutMillis", required = false) Integer timeoutMillis) throws Throwable {
+        return invokeJob(tenant, jobName, block, timeoutMillis, null);
+    }
 
+    @RequestMapping(method = RequestMethod.POST, produces = "application/json", 
+            consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    @ResponseBody
+    @RolesAllowed(SecurityRoles.SCHEDULE_EDITOR)
+    public Map<String, String> invokeJob(@PathVariable("tenant") final String tenant,
+            @PathVariable("job") String jobName, @RequestParam(value = "block", required = false) Boolean block,
+            @RequestParam(value = "timeoutMillis", required = false) Integer timeoutMillis,
+            @RequestParam(value = "overrideProperties") final MultipartFile overrideProperties) throws Throwable {
```

Comment:
Do you mean the body of the request should contain plain text which is can then be parsed as properties?